packages/rancher-monitoring/generated-changes/overlay/files/rancher/home/rancher-default-home-with-windows.json

```diff
@@ -99,7 +99,7 @@
         "tableColumn": "",
         "targets": [
           {
-            "expr": "(1 - avg(irate(node_cpu_seconds_total{mode=\"idle\"}[5m]))) * 100",
+            "expr": "(1 - avg(irate({__name__=~\"node_cpu_seconds_total|windows_cpu_time_total\",mode=\"idle\"}[5m]))) * 100",
```

Comment:
Does this regex evaluate _either name_? I have this question for the following examples as well (in the rest of this file).

Reply:
Yes. Only one or the other will ever be true for any node and the labels we are querying for for either series is exactly the same.

Reply:
If the labels are different, then we need to actually separate them as two series and do the whole `OR on() vector(0)` in order to prevent `No Data` from showing up (for example, when there isn't any Windows nodes currently on a Windows cluster). The filesystem usage metrics are an example of that.